content/docs/12-enterprise-version/03.5-helm-chart-install-reference.md

```diff
@@ -0,0 +1,261 @@
+---
+title: "Helm Chart Install Reference"
+metaTitle: "Helm Chart Install References"
+metaDescription: "Reference page for Pallete Helm chart self-installation values.yaml file parameters."
+icon: ""
+hideToC: False
+fullWidth: False
+---
+
+# Helm Chart Install Reference 
+
+You can use the Palette Helm Chart to install Palette in a multi-node Kubernetes cluster in your production environment. The Helm chart allows you to customize values in the values.yaml file. This reference guide lists and describes parameters available in the values.yaml file from the Helm Chart for your installation. To learn how to install Palette using the Helm Chart, refer to [Helm Chart Mode](https://docs.spectrocloud.com/enterprise-version/deploying-palette-with-helm).  
+
+# Required parameters
+
+The following parameters are required:
+
+- **env.rootDomain**- Used to configure the domain for the Palette installation. **Note:** You should create a DNS record separately, and it should be a wildcard to account for Organization prefixes. Review the [Environment parameters](#environment) to learn more. 
```

Comment:
```suggestion
- **env.rootDomain**- Used to configure the domain for the Palette installation.  You should create a CNAME DNS record separately, and it should be a wildcard to account for Organization prefixes. Review the [Environment parameters](#environment) to learn more. 
```

Let's remove Note:, just add it into the description otherwise it looks odd.